user_management/src/user/urls.py

```diff
@@ -1,10 +1,16 @@
 from django.urls import path
 
-from .views import IsUsernameTakenView, RefreshJWT, SignInView, SignUpView
+from .views import (CheckForgotPasswordCodeView,
+                    ForgotPasswordChangePasswordView, ForgotPasswordView,
+                    IsUsernameTakenView, RefreshJWT, SignInView, SignUpView)
 
 urlpatterns = [
     path('signup/', SignUpView.as_view(), name='signup'),
     path('signin/', SignInView.as_view(), name='signin'),
     path('refresh-jwt/', RefreshJWT.as_view(), name='refresh-jwt'),
     path('username-exist/', IsUsernameTakenView.as_view(), name='username-exist'),
+    path('forgot-password/send-code/', ForgotPasswordView.as_view(), name='forgot-password-send-code'),
+    path('forgot-password/check-code/', CheckForgotPasswordCodeView.as_view(), name='forgot-password-check-code'),
```

Comment:
Shouldn't you rename the view to `ForgotPasswordCheckCodeView` ?

Reply:
d51e53c